styles.css

```diff
@@ -458,3 +460,399 @@ form {
+.about-project-desktop,
+.client-name-desktop,
+.name-desktop,
+.li-desktop,
+.project-desktop,
+.menu-logo {
+  display: none;
+}
+
+@media only screen and (max-width: 1440px) and (min-width: 768px) {
+  /* headliner page */
+  .toolbar {
+    padding-left: 10%;
+    width: 90vw;
+    margin-right: 0;
+  }
+
+  .toolbar-links {
+    color: #344563;
+    text-decoration: none;
+    display: flex;
+    justify-content: start;
+    width: 20vw;
+    gap: 4%;
+    margin-right: 0;
+    padding-right: 0;
+  }
+
+  .toolbar-links > a {
+    padding: 0;
+    margin: 0;
+  }
+
+  #work {
+    height: 320vh;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    background-color: #f7f7f9;
+  }
+
+  .wrapper-1 {
+    height: 100vh;
+    background-color: #f7f7f9;
+  }
+
+  .headline {
+    background-image: url(images/desktop-pictures/Header\ bg.png), url(images/desktop-pictures/header\ background\ shape-desktop.svg);
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: 105.9%;
+    padding: 20%;
+    padding-top: 0;
+    padding-bottom: 0;
+    border-radius: 0 0 0 180px;
+    height: 68vh;
+  }
+
+  .headline > * {
+    padding: 12px;
+  }
+
+  .wrapper-container {
+    margin: 0;
+    padding: 0;
+  }
+
+  /* work section */
+  .project {
+    padding: 10%;
+    padding-bottom: 0;
+    height: 65vh;
+  }
+
+  .container {
+    border: 1px solid #dfe1e6;
+    grid-template-columns: repeat(auto-fill, minmax(50%, 1fr));
+    grid-template-rows: repeat(6, 1fr);
+    grid-gap: 0;
+    height: 53vh;
+    padding: 2rem;
+  }
+
+  .project-1 {
+    order: 3;
+  }
+
+  .project-2 {
+    order: 4;
+    padding-bottom: 10%;
+  }
+
+  .project-3 {
+    order: 1;
+  }
+
+  .project-3 > .container {
+    width: 75vw;
+  }
+
+  .project-4 {
+    order: 2;
+  }
+
+  .container > * {
+    margin: 0;
+    padding-left: 2vw;
+  }
+
+  .container > img {
+    width: 100%;
+    grid-row: 1 / 7;
+    padding: 3%;
+    padding-right: 0;
+  }
+
+  .image-desktop,
+  .li-desktop,
+  .project-desktop,
+  .client-name-desktop,
+  .name-desktop {
+    display: block;
+  }
+
+  .li-mobile,
+  .project-mobile,
+  .client-name-mobile,
+  .image-mobile {
+    display: none;
+  }
+
+  .container > .flip-img {
+    grid-column: 2 / 3;
+  }
+
+  .client-name {
+    height: 50%;
+    align-self: end;
+    margin: 2%;
+    font-size: 2vw;
+  }
+
+  .container > ul {
+    margin: 7%;
+    margin-right: 0;
+    height: 3vh;
+    margin-top: 0;
+    margin-bottom: 0;
+    align-self: center;
+  }
+
+  .about-project {
+    margin: 2%;
+  }
+
+  .buttons {
+    margin: 2%;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .see-project {
+    margin-left: 7%;
+    padding: 0 1% 0 1%;
+    font-size: 1.3vw;
+    width: 10vw;
+    align-self: start;
+  }
+
+  .contact {
+    width: 50%;
+  }
+
+  .logo,
+  .toolbar-links > a,
+  .about,
+  h2,
+  .about-project {
+    font-size: 1.3vw;
+    line-height: 1.7;
+  }
```

Comment:
- [ ] This is close but the navbar links and the logo don't have an underline in the given Figma design so please make sure to remove them as it's causing a discrepancy.
- Your design:
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/dbd85f0a-e75e-41d9-aae2-dfefed9b9161)
- Figma design:
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/a18d60a8-2ac8-4e11-a111-ad858c569633)